### PR TITLE
Add try: except: for connections problem.

### PR DESCRIPTION
**Pull request by jfclere**
missing try: expect:

### CODE REVIEW

File: test-demo-webapp.py [hunk 1]
```diff
@@ -31,11 +31,13 @@ def request_response(self):
   # Return False if sticky True otherwise
```
Comment (jajik) [inline test-demo-webapp.py:31]: Could you please remove the comment / or reword it.

**Review by jajik — COMMENTED** [review]
What should `request_response_loop` return in case of `self.request_response()` returns  `False` but `i < n`? I would expect `False` but currently it returns `None` (?).

**Comment by jfclere** [comment]
Should be better now, thanks.

**Review by jajik — APPROVED** [review]
Looks fine now, sorry for not spotting it the first time.

**Comment by jajik** [comment]
Just please squash it so that there is not merge commit present. Feel free to merge it afterwards.

**Pull request merged**